Kernel/Arch/aarch64/Registers.h

```diff
@@ -834,6 +834,108 @@ struct alignas(u64) SCTLR_EL1 {
 };
 static_assert(sizeof(SCTLR_EL1) == 8);
 
+// https://developer.arm.com/documentation/ddi0601/2025-06/AArch64-Registers/SCTLR-EL2--System-Control-Register--EL2-
+// System Control Register
+struct alignas(u64) SCTLR_EL2 {
```

Comment:
why the `alignas` It already is aligned like a u64 
(Note: `alignas` can only increase alignment IIRC)

Reply:
All register definitions in this file are `alignas(u64)`.
I also think this should be unnecessary, especially since alignment shouldn't matter for writing system registers with `msr/mrs`.

We can clean that up in a future PR.